tests/authors.test.js

```diff
@@ -0,0 +1,137 @@
+const { expect } = require('chai');
+const { Author } = require('../src/models');
+const request = require('supertest');
+const app = require('../src/app');
+
+describe('/authors', () => {
```

Comment:
Indentation is inconsistent again, it should be uniform across the whole project